airbyte-cdk/python/airbyte_cdk/sources/declarative/async_job/job.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+
+
+from datetime import timedelta
+from typing import Optional
+
+from airbyte_cdk import StreamSlice
+from airbyte_cdk.sources.declarative.async_job.timer import Timer
+
+from .status import AsyncJobStatus
+
+
+class AsyncJob:
+    """
+    Note that the timer will only stop once `update_status` is called so the job might be completed on the API side but until we query for
```

Comment:
can you add a description of what the class's purpose is in addition to the note?